extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -42,28 +40,56 @@
 
   @JsonProperty("productUrl")
   private String productUrl;
-  
-  public String getId() { return id; }
 
-  public String getDescription() { return description; }
+  public String getId() {
+    return id;
+  }
+
+  public String getDescription() {
+    return description;
+  }
+
+  public String getBaseUrl() {
+    return baseUrl;
+  }
 
-  public String getBaseUrl() { return baseUrl; }
+  public String getMimeType() {
+    return mimeType;
+  }
 
-  public String getMimeType() { return mimeType; }
+  public String getFilename() {
+    return filename;
+  }
 
   public String getProductUrl() {
     return productUrl;
   }
 
-  public MediaMetadata getMediaMetadata() { return mediaMetadata; }
+  public MediaMetadata getMediaMetadata() {
+    return mediaMetadata;
+  }
+
+  public void setDescription(String description) {
+    this.description = description;
+  }
 
-  public void setDescription(String description) { this.description = description; }
+  public void setBaseUrl(String baseUrl) {
+    this.baseUrl = baseUrl;
+  }
 
-  public void setBaseUrl(String baseUrl) { this.baseUrl = baseUrl; }
+  public void setId(String id) {
+    this.id = id;
+  }
 
-  public void setId(String id) { this.id = id; }
+  public void setMimeType(String mimeType) {
+    this.mimeType = mimeType;
+  }
 
-  public void setMimeType(String mimeType) { this.mimeType = mimeType; }
+  public void setFilename(String filename) {
```

Comment:
This is an example of the difficulty it causes. This setFilename method has been added but it is hard to spot amongst all the formatting changes.